src/utils/index.ts

```diff
@@ -17,8 +17,8 @@ export function createObjectFromLocalStorage(key: string) {
     return undefined;
 }
 
-export function isValidProp(p: number) {
-    return p !== undefined && !isNaN(p); // eslint-disable-line no-restricted-globals
+export function getStringValue(p: number | string) {
+    return (typeof p === 'string') ? p : p.toString();
```

Comment:
```suggestion
    return (typeof p == 'string') ? p : p.toString();
```

Reply:
thank for the suggestion, but I believe we don’t need here type coercion.. '===' is working fine